backend-rust/src/graphql_api/transaction_metrics.rs

```diff
@@ -0,0 +1,52 @@
+use async_graphql::{Context, Object, SimpleObject};
```

Comment:
We could also consider making this a more general `transactions` module and include the already-existing transaction-related queries from `graphql_api.rs`.